FlightDynamics.m

```diff
@@ -1,81 +1,93 @@
 clc; clear; close all;
 
-global windupper;       %declare upper wind speed

```

Comment:
Renaming Global variables to two Structs name Rocket and Atmos.
Rocket contains rocket related parameters
Atmos contains atmosphere related parameters

Goal is to reduce bugginess and speed of the simulator when we expand to Monte Carlo sims, as well general coding practice discourages the use of Globals: 
https://www.mathworks.com/matlabcentral/answers/369269-global-variable-not-working-in-the-matlab-workspace